### DISCUSSION TITLE
Make MenuBar for Desktop Apps use the macOS Top Menubar, instead of the custom made MenuBar

### DISCUSSION THREAD

**Discussion by alexrjs** [flet-dev/flet#3016] [Ideas]
I noticed that the flet "MenuBar" is custom and doesn't use the macOS Top Menubar for flet Desktop Apps. Would be nice if there would be a way to use the Top Menubar of macOS and not the Custom flet "MenuBar". From an UI perspective the flet "MenuBar" is very unusual for macOS flet Desktop Apps.

**Comment by ndonkoHenri** [comment]
Subscribe to this issue: https://github.com/flet-dev/flet/issues/285
